### DISCUSSION TITLE
Licensing problems (CPU licence)

### DISCUSSION THREAD

**Discussion by btuck812** [MPh-py/MPh#74] [General]
Hi, following instructions on calling a basic COMSOL model on Windows using the stand-alone client, I came across two different licensing issues - one related to 3D geometry requiring the SOLIDWORKS LiveLink (oddly) and another related to the CPU-only licence not covering remote use. I've tested the Java-COMSOL interface itself, and that works perfectly - I was wondering if you had any insight into this or if this is a known COMSOL issue. Any help would be greatly appreciated, I have also extended this question to COMSOL support.

Thanks!

**Comment by john-hen** [comment]
Remote use is only covered by the network license, not the "single CPU" (or whatever it's called). That's the only thing that comes to mind. Other than that, I've never encountered these problems, nor would I know how to reproduce them. Yes, Comsol Support is the better place to ask when it comes to license issues. MPh simply assumes you have all the licenses you need.

**Reply by btuck812** [reply]
Thank you for responding so quickly! The issue here is that I'm not using remote use, I'm using the computer COMSOL is installed on - yet when running the MPh script, it claims that I am trying to access COMSOL remotely (which I don't quite understand)

**Reply by john-hen** [reply]
If you have a specific error to report, ideally along with the script that (re)produces the error, then I can take a look. But please note that the MPh test suite (which is part of the repo here on GitHub) is routinely run, on Windows as well as Linux, using the exact same license you have: single-CPU. So for all I know, this works as expected, and I have nothing else to go on.
